test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/RepresenteeControllerSpec.scala

```diff
@@ -1662,7 +1893,113 @@ class RepresenteeControllerSpec
         }
       }
     }
+    "handling requests for check your answers page" must {
```

Comment:
there should be an existing part of the spec which tests the cya endpoint which you can put your tests in

Reply:
Unfortunately no - if I don't miss, checked twice, except the one I've created.

Reply:
https://github.com/hmrc/cgt-property-disposals-frontend/blob/master/test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/RepresenteeControllerSpec.scala#L1565
^^^ I was thinking around there

Reply:
Done.